<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-1-g15f4949  Build Version: 1.6.7594.29634
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="Athena.SLDASM">
  <link
    name="link0">
    <inertial>
      <origin
        xyz="0.00084199 6.0516E-06 0.027312"
        rpy="0 0 0" />
      <mass
        value="1.07" />
      <inertia
        ixx="0.0017764"
        ixy="1.0653E-09"
        ixz="3.2188E-06"
        iyy="0.0017612"
        iyz="-3.7721E-10"
        izz="0.0034343" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link0.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link0.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="link1">
    <inertial>
      <origin
        xyz="-6.61629444295814E-06 -1.91927132648169E-11 -0.0346908666411378"
        rpy="0 0 0" />
      <mass
        value="0.252237214685977" />
      <inertia
        ixx="0.000171346755451675"
        ixy="7.4094509508841E-10"
        ixz="5.66278592678419E-09"
        iyy="0.000157397378418575"
        iyz="4.22996912934034E-13"
        izz="0.000271224838861379" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint1"
    type="revolute">
    <origin
      xyz="0 0 0.093"
      rpy="0 0 0" />
    <parent
      link="link0" />
    <child
      link="link1" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="link2">
    <inertial>
      <origin
        xyz="-0.139412624204607 0.0189971379530124 -1.67078320598128E-06"
        rpy="0 0 0" />
      <mass
        value="0.401527582473005" />
      <inertia
        ixx="0.000207716029858032"
        ixy="4.20481255217659E-05"
        ixz="-7.27030178177206E-08"
        iyy="0.000448970518181385"
        iyz="1.5330359269074E-08"
        izz="0.000355250815491493" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint2"
    type="revolute">
    <origin
      xyz="0 0 0"
      rpy="1.5708 1.3974 -3.1416" />
    <parent
      link="link1" />
    <child
      link="link2" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="link3">
    <inertial>
      <origin
        xyz="3.71656731372738E-05 0.10004809585708 -1.47541066351612E-06"
        rpy="0 0 0" />
      <mass
        value="0.321758438792249" />
      <inertia
        ixx="0.000251321263865407"
        ixy="-1.13081500697256E-07"
        ixz="-5.10392396225503E-10"
        iyy="0.000176007934532394"
        iyz="9.63782301852364E-08"
        izz="0.000175558662776219" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint3"
    type="revolute">
    <origin
      xyz="-0.2203 0 0"
      rpy="0 0 2.9682" />
    <parent
      link="link2" />
    <child
      link="link3" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="link4">
    <inertial>
      <origin
        xyz="1.1550676970451E-08 -0.00135269732140609 -0.0319158088029138"
        rpy="0 0 0" />
      <mass
        value="0.139042978114618" />
      <inertia
        ixx="8.36795313833226E-05"
        ixy="1.30211755304055E-09"
        ixz="1.43924770203339E-09"
        iyy="4.2816356420825E-05"
        iyz="-4.00657119158233E-06"
        izz="6.52007851384592E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint4"
    type="revolute">
    <origin
      xyz="0 0.228 0"
      rpy="-1.5708 3.1416 0" /><!-- changed p: 0->3.1416 -->
    <parent
      link="link3" />
    <child
      link="link4" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="link5">
    <inertial>
      <origin
        xyz="4.3224E-05 0.017037 0.0058889"
        rpy="0 0 0" />
      <mass
        value="0.1498" />
      <inertia
        ixx="4.591E-05"
        ixy="-3.7679E-10"
        ixz="1.0646E-09"
        iyy="6.5982E-05"
        iyz="5.8437E-07"
        izz="3.0968E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link5.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link5.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint5"
    type="revolute">
    <origin
      xyz="0 0 0"
      rpy="1.5708 0 3.1416" />
    <parent
      link="link4" />
    <child
      link="link5" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="link6">
    <inertial>
      <origin
        xyz="-0.0072971 -2.4348E-05 0.092179"
        rpy="0 0 0" />
      <mass
        value="0.35726" />
      <inertia
        ixx="0.00015096"
        ixy="-1.5249E-07"
        ixz="3.1349E-06"
        iyy="0.0001375"
        iyz="3.6769E-10"
        izz="0.00013855" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link6.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="link6.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="joint6"
    type="revolute">
    <origin
      xyz="0 0 0"
      rpy="1.5708 3.1416 3.1416" /><!-- changed p: 0->3.1416 -->
    <parent
      link="link5" />
    <child
      link="link6" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="10"
      velocity="10" />
  </joint>
  <link
    name="link_eef">
    <inertial>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
      <mass value="0.0"/>
      <inertia ixx="0.0" ixy="0.0" ixz="0.0" iyy="0.0" iyz="0.0" izz="0.0"/>
    </inertial>
    <collision>
      <geometry>
        <cylinder radius="0.0" length="0.0"/>
      </geometry>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
    </collision>
  </link>
  <joint
    name="joint_eef"
    type="fixed">
    <!-- eef on ball joint -->
    <!-- <origin
      xyz="0.0 0.0 0.0"
      rpy="0 -1.5708 0" /> -->
    <!-- eef on gripper center -->
    <!-- <origin
      xyz="0.080 0.0 0.195"
      rpy="0 -1.5708 0" /> -->
    <!-- eef on gripper center z-axis + 0.060m (consider teleop tag)  -->
    <origin
      xyz="-0.060 0.0 0.120"
      rpy="3.1416 -1.5708 0" />
    <parent
      link="link6" />
    <child
      link="link_eef" />
  </joint>
</robot>